pkgs/misc/emulators/wine/sources.nix

```diff
@@ -50,7 +50,7 @@ in rec {
     inherit (unstable) version;
     # FIXME update winestaging sources, when 3.3 is released
```

Comment:
I suppose these FIXME lines can go now as well?

Reply:
sorry, gonna redo it now ;)